bundle/tests/git_test.go

```diff
@@ -14,13 +14,15 @@ import (
 
```

Comment:
Can you move all unit tests to a file called `load_git_details_test.go` next to `load_git_details.go`? Since it's no longer part of the load function, we can just test the mutator.

Reply:
See comment below - I actually think it's good to include load phase here, since it affects Bundle and thus can have effect on this mutator's behaviour.